tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Core.cs

```diff
@@ -166,7 +166,21 @@ internal override bool IsBlocked
             }
         }
 
-        internal override int StatusCode => Context.Response.StatusCode;
+        internal override int? StatusCode
+        {
+            get
```

Comment:
Should IsAdditiveContextDisposed() and SetAdditiveContextDisposed() be used in this PR as well in the same way as https://github.com/DataDog/dd-trace-dotnet/pull/6592?

Reply:
Applied in b309883 😄